rfcs/0000-policy_target_scope.md

```diff
@@ -0,0 +1,267 @@
+applies to at this stage. All traffic to `foo-route` is now painted `green`, while any further routes
+attached to `my-gw` would still be painted `blue`.
+
+>**Note:** The status block of a policy should reflect whether it is shadowing or being shadowed by other
+>policies at that level of the hierarchy.
+
+Which would effectively disable all rate limiting traffic to the `foo-route` and ultimately the `foo`
+`Backend`.
+
+Using another example, let's consider a single `HTTPRoute` with two `HTTPRouteRule`s:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: bar-route
+spec:
+  parentRefs:
+  - name: my-gw
+  hostnames:
+  - "bar.example.com"
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /login
+    backendRefs:
+    - name: bar-svc
+      port: 8081
+  - matches:
+    - headers:
+      - type: Exact
+        name: env
+        value: production
+    backendRefs:
+    - name: bar-svc
+      port: 8080
+```
+
+It is important to consider that only _one_ rule will ever be match for any given requests. It makes for
+a simpler model to have only one _Policy_ to also only ever being matched, aligning the both behaviors.
+To apply a policy on the first rule only, the user needs to duplicate that match on the policy targeting
+`bar-route`:
+
+```yaml
+kind: PaintPolicy
+spec:
+  default:
+    color: red
+  targetRef:
+    kind: HTTPRoute
+    name: foo-route
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /login
+```
+
+Hereby we are coloring the traffic to our `/login` on port `8081` to be `red`. That coloring _will not_
+apply to other matches. So that traffic to `bar-svc` on port `8080` would be colored `green` as per the
+`PaintPolicy` attached to `my-gw`.
+
+The resolving of _Policies_ to enforce occurs at the `HTTPRoute` or within individual `HTTPRouteRule`
+within a route. The merging strategy will resolve the individual `HTTPRouteRule` all the policies boil
+down to, stack them according to the `default` and `override` and appy the most specific one according to
+the Gateway API Policy Attachment hierarchy. Targeting a complete `HTTPRoute` is the equivalent of
+targeting each indidividual `HTTPRouteRule`s that composes the route.
+
+## What actual use-cases does this address?
+
+ - An admin restricts all traffic using a `default` at the gateway to force application developers to
+   "think" about how to authorize access to their services and have them come up with sensible `default`
+   on their `HTTPRoute`s
+ - An application developer protects its service by some rate limit, while an cluster admin could come
+   and use an `override` to mitigate a DDoS.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+So there are a couple of things to consider here:
+
+- status reporting on "not yet" existing `HTTPRouteRule`, e.g. a Policy declares a set of `matches` that
```

Comment:
Yes I think reporting status in the RLP and in the HTTPRoute / Gateway is worth consideration
example on a condition of the HTTPRoute status we could add something like ```GVK name namspace enforced```